src/core_scan/Report_pro_findings.mli

```diff
@@ -0,0 +1,4 @@
+val annotate_pro_findings :
```

Comment:
Small comment please. Maybe you can say what it currently does, which is annotate as pro
findings whose trace cross files, and maybe add some TODOs for other stuff we could do in the futur.